aidants_connect_web/tests/test_views/test_service.py

```diff
@@ -129,8 +134,124 @@ def test_environment_variables_are_accessible(self):
         self.assertEqual(secret_key, "Everything is awesome")
 
 
-@tag("service")
+@tag("service", "this")
```

Comment:
Pas sûr de piger la sémantique de `this` ici 🤔

Reply:
C'est un artefact de mon process de dev où je tag en "this" ou "hère" le test que je veux faire passer.